docs/keyboards/ergonaut-one/build-guide.md

```diff
@@ -41,21 +56,39 @@ Before soldering the Xiao, please, apply a **small** amount of solder on the bat
 
 #### Diodes
 
-If you are building a "Bottom Xiao" configuration, diodes should be on the **SAME** side as Xiao.
-If you are building a "Top Xiao" configuration, diodes should be on the **OPPOSITE** side from Xiao.
+"Bottom Xiao" configuration: diodes should be on the **SAME** side as Xiao.  
+"Top Xiao" configuration: diodes should be on the **OPPOSITE** side from Xiao.
 
-Please take note that diodes have polarity.
+Please take note that diodes have polarity that are shown on the PCB and on the
+diodes with a line.
 
 ![](/img/one_build_guide/pcb/diode.png)
 
+![](/img/one_build_guide/pcb/diode_2.jpg)
+
+1. Add a small amount of solder to one pad
+2. Place the diode on the pads with tweezers, making sure they are in the
+   correct orientation
+3. Melt the solder already on the pad and ensure the diode aligned properly
+4. Solder the other pad
+
 | Left                                  | Right                                 |
 | ------------------------------------- | ------------------------------------- |
 | ![](/img/one_build_guide/pcb/3_l.jpg) | ![](/img/one_build_guide/pcb/3_r.jpg) |
 
 #### Hotswap Sockets
 
-If you are building a "Bottom Xiao" configuration, sockets should be on the **SAME** side as Xiao.
-If you are building a "Top Xiao" configuration, sockets should be on the **OPPOSITE** side from Xiao.
+"Bottom Xiao" configuration: sockets should be on the **SAME** side as Xiao.  
+"Top Xiao" configuration: sockets should be on the **OPPOSITE** side from Xiao.
+
+The orientation of the sockets does **not** matter.
```

Comment:
I'm still uncertain on this if the orientation actually matters or not for these sockets. I couldn't find any resources about the correct orientation, but just followed the images in the guide that needed zooming in to check.

Confirmation on if this is accurate or not would be great 😃